docs/content/preview/migrate/_index.md

```diff
@@ -22,14 +22,13 @@ YugabyteDB Voyager manages the entire lifecycle of a database migration, includi
 
 - Free and completely open source.
 - Supports widely used databases for migration and doesn't require changes to the [source databases](#source-databases) in most cases.
-- Supports all YugabyteDB products (v2.12 and above) as the [target database](#target-database).
+- Supports all YugabyteDB products (YugabyteDB stable versions 2.14.5.0 and later, and preview versions 2.17.0.0 and later.) as the [target database](#target-database).
```

Comment:
I believe its okay not to add 2.16.0.0 because we are mentioning *stable* versions 2.14.5.0 and higher and 2.16.0.0  comes as stable. Will confirm with Alex again and address it if needed. Thanks.